### PR TITLE
libcurl is now optional. Updated to the latest versions of libcurl and expat.

### PR DESCRIPTION
**Pull request by LibreShawn**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -9,11 +9,13 @@ class Exiv2Conan(ConanFile):
                'xmp': [True, False],
                'iconv': [True, False],
                'webready': [True, False],
+               'libcurl': [True, False],
               }
     default_options = ('unitTests=True',
                        'xmp=False',
                        'iconv=False',
                        'webready=False',
```
Comment (kmilos) [inline conanfile.py:17]: Need to have webready switched on as well, otherwise curl option is ignored

Comment (LibreShawn) [reply conanfile.py:17]: If libcurl is only used when webready is enabled, perhaps I don't need a libcurl option and should change the code to only require libcurl if webready is enabled. The problem I'm trying to solve is that conan is building libcurl and its dependencies (OpenSSL) when I don't need it, and that takes a long time.

Comment (kmilos) [reply conanfile.py:17]: Yes, or leave libcurl option on all the time and toggle just webready instead, and install libcurl only if webready is enabled...

FYI, this is coming from here:

https://github.com/Exiv2/exiv2/blob/5dca23bddcfa69abc3f486df65f03b344ee53bec/cmake/findDependencies.cmake#L25-L29
https://github.com/Exiv2/exiv2/blob/5dca23bddcfa69abc3f486df65f03b344ee53bec/src/CMakeLists.txt#L200-L207

Comment (LibreShawn) [reply conanfile.py]: Good info, thanks.

I pushed an update to my repo, but I'm not sure what I have to do to get that to show up here. I got rid of the libcurl option and only require libcurl if webready is enabled.

**Review by kmilos — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**